sites/partners/src/components/listings/PaperListingForm/sections/ApplicationTypes.tsx

```diff
@@ -32,11 +34,20 @@ interface Methods {
 const ApplicationTypes = ({ listing }: { listing: FormListing }) => {
   // eslint-disable-next-line @typescript-eslint/unbound-method
   const { register, setValue, watch, errors, getValues } = useFormContext()
+  const { doJurisdictionsHaveFeatureFlagOn } = useContext(AuthContext)
+
   // watch fields
   const digitalApplicationChoice = watch("digitalApplicationChoice")
   const commonDigitalApplicationChoice = watch("commonDigitalApplicationChoice")
   const paperApplicationChoice = watch("paperApplicationChoice")
   const referralOpportunityChoice = watch("referralOpportunityChoice")
+
+  const disableCommonApplication = doJurisdictionsHaveFeatureFlagOn(
+    FeatureFlagEnum.disableCommonApplication,
+    listing?.jurisdictions?.id,
```

Comment:
issue: this doesn't work if it's a new listing since there won't be a `listing`. I think you should use a `watch` of the `jurisdictions` to get the correct id.